nautobot/dcim/tests/test_api.py

```diff
@@ -83,12 +83,20 @@ def test_trace(self):
                 location=Location.objects.filter(location_type=LocationType.objects.get(name="Campus")).first(),
                 device_type=DeviceType.objects.first(),
                 role=Role.objects.get_for_model(Device).first(),
+                status=Status.objects.get_for_model(Device).first(),
                 name="Peer Device",
             )
             if self.peer_termination_type is None:
                 raise NotImplementedError("Test case must set peer_termination_type")
-            peer_obj = self.peer_termination_type.objects.create(device=peer_device, name="Peer Termination")
-            cable = Cable(termination_a=obj, termination_b=peer_obj, label="Cable 1")
+            if self.peer_termination_type is Interface:
```

Comment:
Only `Interface` has a `status` huh? interesting.

Reply:
VMInterface does too, but it can't be cabled.